com.github.bharatkalluri.gifup.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "com.github.bharatkalluri.gifup",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.bharatkalluri.gifup",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=home",
+        "--talk-name=ca.desrt.dconf"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.la",
+        "/share/gir-1.0",
+        "/lib/girepository-1.0",
+        "/share/vala",
+        "/share/gtk-doc"
+    ],
+    "modules" : [
+        {
+            "name" : "ffmpeg",
+            "config-opts" : [
+                "--enable-shared",
+                "--disable-static",
+                "--disable-doc",
+                "--disable-ffplay",
+                "--disable-ffserver",
+                "--disable-devices",
+                "--enable-gnutls",
+                "--enable-libvorbis"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://ffmpeg.org/releases/ffmpeg-3.4.2.tar.xz",
+                    "sha256" : "2b92e9578ef8b3e49eeab229e69305f5f4cbc1fdaa22e927fc7fca18acccd740"
+                }
+            ],
+            "post-install" : [
+                "install -Dm644 COPYING.LGPLv3 /app/share/licenses/ffmpeg/COPYING"
+            ],
+            "cleanup" : [
+                "/share/ffmpeg"
+            ]
+        },
+        {
+            "name" : "gifup",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
```

Comment:
Flathub only hosts reproducible builds. So you need either a release tarball or a specific commit here.

Reply:
Added a commit hash now